### ISSUE TITLE
Cant Upload a file in fb or any other site

### ISSUE THREAD

**Issue by hkk007** [anthonycr/Lightning-Browser#405]
I  have downloaded sample apk and i cant upload a file....nothing happens when i press choose file button. It is a must have feature... iam using android 4.4.2  kitkat

**Comment by bebolint98** [comment]
Yes, same here on android 4.4.4 and if I put the phone to landscape and back to portrait mode the space where the keyboard is, it stay here.

**Comment by bebolint98** [comment]
Yes, same here on android 4.4.4 and if I put the phone to landscape and back to portrait mode the space where the keyboard is, it stay here.

**Comment by bebolint98** [comment]
Yes, same here on android 4.4.4 and if I put the phone to landscape and back to portrait mode the space where the keyboard is, it stay here.

**Comment by bebolint98** [comment]
Yes, same here on android 4.4.4 and if I put the phone to landscape and back to portrait mode the space where the keyboard is, it stay here.
